src/vm/gdbjit.cpp

```diff
@@ -1154,19 +1155,47 @@ void TypeMember::DumpStaticDebugInfo(char* ptr, int& offset)
         memberEntry.m_member_specification = m_member_offset;
         memcpy(ptr + offset, &memberEntry, sizeof(DebugInfoStaticMember));
 
-        char buf[ptrSize + 2] = {0};
-        buf[0] = ptrSize + 1;
-        buf[1] = DW_OP_addr;
-
-        for (int i = 0; i < ptrSize; i++)
+        // for value type static fields compute address as:
+        // addr = (*addr+sizeof(OBJECTREF))
+        if (m_member_type->GetTypeHandle().GetSignatureCorElementType() ==
```

Comment:
I am not sure if this is the case for all static fields of value types, based on the comments in corinfo.h (https://github.com/dotnet/coreclr/blob/master/src/inc/corinfo.h#L117-L154).
@jkotas, can you please comment on this?